firestore/query.go

```diff
@@ -1036,3 +1045,72 @@ func (it *btreeDocumentIterator) next() (*DocumentSnapshot, error) {
 }
 
 func (*btreeDocumentIterator) stop() {}
+
+// AggregationQuery allows for generating aggregation results of an underlying
+// basic query. A single AggregationQuery can contain multiple aggregations.
+type AggregationQuery struct {
+	aggregateQueries []*pb.StructuredAggregationQuery_Aggregation // aggregateQueries contains all of the queries for this request.
```

Comment:
nit: comments above fields makes for better line lengths and easier-to-parse code

Reply:
Done.